Update 2020-02-07-the-ember-times-issue-134.md (#518)
ember-learn/ember-blog: source/2020-02-07-the-ember-times-issue-134.md

---
title: The Ember Times - Issue No. XXS
author: Anne-Greeth van Herwijnen, the crowd
tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
alias : "blog/2020/xx/xx-the-ember-times-issue-XXX.html"
responsive: true
---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

How structuring your Figma can make your developer life easier 🎨,

READMORE

---

## [From Figma to Ember to Storybook … with 2 Clicks 🎨](https://gos.si/blog/from-figma-to-ember-to-storybook-with-2-clicks/)

> [Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has just been publish and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .

Reviewer: To make both the markdown linter happy and to round up the sentence, I think it'd be:

```suggestion

[Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has just been published and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .
```

What do you think?

> Part 2 on design tokens in Ember and part 3 on documentation within the Ember Ecosystem are on their way. Follow [@unistyler](https://twitter.com/unistyler) on Twitter to keep posted on when they are published. 

Reviewer 2: I think "be notified" sounds better here. You'd keep someone posted when you are the person who makes an announcement. :)
```suggestion
Part 2 on design tokens in Ember and part 3 on documentation within the Ember Ecosystem are on their way. Follow [@unistyler](https://twitter.com/unistyler) on Twitter to be notified when they are published. 
```

---
title: The Ember Times - Issue No. XXS
author: Anne-Greeth van Herwijnen, the crowd
tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
alias : "blog/2020/xx/xx-the-ember-times-issue-XXX.html"
responsive: true
---

<SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹

How structuring your Figma can make your developer life easier 🎨,

READMORE

---

## [From Figma to Ember to Storybook … with 2 Clicks 🎨](https://gos.si/blog/from-figma-to-ember-to-storybook-with-2-clicks/)

> [Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has just been published and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .

Reviewer 2: ```suggestion
[Thomas Gossmann @gossi](https://github.com/gossi) has started a blog post series on Figma, Ember and Storybook. It is a 3 part series. [Part 1](https://gos.si/blog/full-featured-themes-in-figma/) has been published and will tell you all about how to organise your design in Figma. So if you want to learn something about themes and design tokens, go ahead and read  [Part 1: Full Featured Themes in Figma](https://gos.si/blog/full-featured-themes-in-figma/) .
```
